### ISSUE TITLE
[Feature]: Add better syntax highlighting to ComponentPreview

### ISSUE THREAD

**Issue by Saeid-Za** [radix-vue/shadcn-vue#372]
### Describe the feature

There is currently a black & white theme for code highlighting in Documentation, I'd suggest changing this to a more vibrent theme, like the current theme used in radix-vue documents.

### Additional information

- [X] I intend to submit a PR for this feature.
- [ ] I have already implemented and/or tested this feature.

**Comment by sadeghbarati** [comment]
I agree with that but I thought it would be better to be like main shadcn-ui that's why I didn't use shiki themes

**Comment by Saeid-Za** [comment]
I understand your point, but with the Vue ecosystem, we have the ability to incorporate features that are not available in shadcn-ui (the React version). We are not limited to only support what  shadcn-ui supports.

I have cloned and modified shadcn-vue to suit my own needs, and some of the changes I've made could benefit the end-users that are not currently implemented in shadcn-ui. I am working on creating some feature improvement requests based on my experiments.

Regarding this particular feature request, I may have a bias towards certain styles. If you believe that the current styling is better, feel free to close this issue as resolved. ❤️
